pkg/webhooks/reconciller.go

```diff
@@ -71,6 +72,7 @@ func (reconciler *MutatingWebhookReconciler) Reconcile(ctx context.Context, clie
 	}
 
 	_, err := controllerutil.CreateOrUpdate(ctx, client, cr, func() error {
+		ownerutil.EnsureOwner(cr, owner)
```

Comment:
I suppose RHMI CR would act as owner here.
We should not set that because `MutatingWebhookConfiguration` [is not namespaced](https://github.com/kubernetes/api/blob/dccc90724807c47108487bab2c317ece185246f5/admissionregistration/v1beta1/types.go#L153-L162),
and non-namespaced(cluster scoped) object can not be owned by namespaced objects(e.g. RHMI), as per docs:
```
Note: Cross-namespace owner references are disallowed by design. This means: 
1) Namespace-scoped dependents can only specify owners in the same namespace, and owners that are cluster-scoped. 
2) Cluster-scoped dependents can only specify cluster-scoped owners, but not namespace-scoped owners.
```
From: https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/